operations/helm/charts/mimir-distributed/templates/ingester/ingester-svc.yaml

```diff
@@ -25,6 +25,9 @@ spec:
   {{- if semverCompare ">= 1.22-0" (include "mimir.kubeVersion" .) }}
   internalTrafficPolicy: {{ .Values.ingester.service.internalTrafficPolicy }}
   {{- end }}
+  {{- if .Values.ingester.service.trafficDistribution }}
+  trafficDistribution: {{ .Values.ingester.service.trafficDistribution }}
```

Comment:
this is not used for any of the data transfer for reads or writes. IIRC the service is only used for some admin endpoints like TSDB statistics. Can you clarify this in the values.yaml?